rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -831,6 +843,233 @@ impl Operation for GetTime {
     }
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct GetCanisterHttp;
+
+fn http_method_from(
+    http_method: &ic_types::canister_http::CanisterHttpMethod,
+) -> CanisterHttpMethod {
+    match http_method {
+        ic_types::canister_http::CanisterHttpMethod::GET => CanisterHttpMethod::GET,
+        ic_types::canister_http::CanisterHttpMethod::POST => CanisterHttpMethod::POST,
+        ic_types::canister_http::CanisterHttpMethod::HEAD => CanisterHttpMethod::HEAD,
+    }
+}
+
+fn http_header_from(
+    http_header: &ic_types::canister_http::CanisterHttpHeader,
+) -> CanisterHttpHeader {
+    CanisterHttpHeader {
+        name: http_header.name.clone(),
+        value: http_header.value.clone(),
+    }
+}
```

Comment:
Is this a result from the copied types? Why can't we import them?

Reply:
> Why can't we import them?

We can't really import them since
- we'd obtain a dependency on `/rs` in `/packages`
- we couldn't derive `JsonSchema`